students/tbrackney/Lesson7/html_render.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+"""
+File Name: html_render.py
+Author: Travis Brackney
+Class: Python 201 - Self paced online
+Date Created 5/12/2018
+Python Version: 3.6.4
+"""
+
+''' '''
+
+
+class Element():
+    """Class for rendering html page"""
+    tag = None
+    i_string = '    '
+    indent = 0
+    new_lines = True
+    keywords = ''
```

Comment:
I don' t think you need to define these last three as class attributes. You don't appear to be using the `indent` class attribute, but rather passing the value in as an instance method parameter to your tag and render methods. The `new_lines` attribute could be a method level variable inside the render method, since you're overriding render in the SelfClosing and OneLine subclasses. And you're setting `keywords` as an instance attribute in the `__init__` method which is more appropriate than making it class level. Remember class attribute are for values that will be shared between all instances of the class.